mintpy/closure_phase_bias.py

```diff
@@ -8,56 +8,128 @@
     parser.add_argument('--nl', dest = 'nl', type = int, default = 20, help = 'connection level that we are correcting to (or consider as no bias)')
-    parser.add_argument('--numsigma',dest = 'numsigma', type = float, default = 3, help = 'Threshold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
-    parser.add_argument('--epi',dest = 'episilon', type = float, default = 0.3, help = 'Threshold for amplitude (0-1), default 0.3')
-    parser.add_argument('--maxMemory', dest = 'max_memory', type = float, default = 8, help = 'max memory to use in GB')
+    parser.add_argument('--bw', dest = 'bw', type = int, default = 10, help = 'bandwidth of time-series analysis that you want to correct')
+    parser.add_argument('--num_sigma',dest = 'num_sigma', type = float, default = 3, help = 'Threashold for phase (number of sigmas,0-infty), default to be 3 sigma of a Gaussian distribution (assumed distribution for the cumulative closure phase) with sigma = pi/sqrt(3*num_cp)')
+    parser.add_argument('--epi',dest = 'episilon', type = float, default = 0.3, help = 'Threashold for amplitude (0-1), default 0.3')
+    parser.add_argument('--noupdate',dest = 'update_cp', action = 'store_false', help = 'Use when no need to compute closure phases')
     parser.add_argument('-o', dest = 'outdir', type = str, default = '.', help = 'output file directory')
+    parser.add_argument('-a','--action', dest='action', type=str, default='create_mask',
+                        choices={'create_mask', 'quick_bias_estimate', 'bias_estimate'},
+                        help='action to take (default: %(default)s):\n'+
+                             'create_mask -  create a mask of areas susceptible to closure phase errors\n'+
+                             'quick_bias_estimate - estimate how bias decays with time, will output sequential closure phase files, and gives a quick and appximate bias estimation\n'
+                             'bias_estimate - estimate how bias decays with time, processed for each pixel on a pixel by pixel basis')
+    parser = arg_group.add_parallel_argument(parser)
+    parser = arg_group.add_memory_argument(parser)
     return parser
 
 def cmd_line_parse(iargs=None):
     parser = create_parser()
     inps = parser.parse_args(args = iargs)
     return inps
 
-# Obtain sum of consecutive complex sequential closure phase of connection n
-def cum_seq_closurePhase(SLC_list, date12_list_all, ifgram_stack, ref_phase, n, box):
+def seq_closure_phase(slc_list, date12_list_all, ifgram_stack, ref_phase, n, box):
+    """
+    Computes wrapped sequential closure phases of conneciton-n
+    Input parameters:
+        slc_list : list of SLC dates
+        date12_list_all: date12 of all the interferograms stored in the ifgramstack file
+        ifgram_stack: stack file
+        ref_phase : phase timeseires of the refernce pixel
```

Comment:
If `ref_phase` is from `ifgramStack.get_reference_phase()`, shouldn't be "unwrap phase of all pairs" instead of "phase time series"?

Reply:
You are right!